client/src/components/input/index.js

```diff
@@ -11,10 +11,11 @@ const Input = ({
   className,
   name,
   type,
-  value
+  value,
+  labelClassName
 }) => (
   <div className="container-input">
-    <label className="label-style">{labelText}</label>
+    <label className={labelClassName}>{labelText}</label>
```

Comment:
Here, you could do the same `classnames` type of fix that you did for the `<Select` component, or another idea is you could have 'label-style' be the default class and use labelClassName as an override - `<label className={labelClassName || 'label-style'}`

Either way, the idea is to save you the trouble of passing "label-style" every time you want to create an Input component.